src/components/PlayWord/PlayWord.test.js

```diff
@@ -32,16 +32,29 @@ describe('on player types a word', () => {
 describe('on button click', () => {
     it('calls sendMessage', () => {
         const spy = jest.spyOn(PlayWord.prototype, 'sendMessage');
-        const wrapper = mount(<PlayWord playWord={() => {}}/>);
+        const wrapper = mount(<PlayWord validateWord={() => {}}/>);
         wrapper.find({ 'data-cy': 'send-word' }).simulate('click');
         expect(spy).toHaveBeenCalled();
         spy.mockRestore();
     });
     it('calls playWord', () => {
```

Comment:
Could you change this test to `it('calls validateWord')`?

Reply:
the method inside the component is called sendMessage though, which calls validateWord